### DISCUSSION TITLE
ScriptReference/RaycastHit-triangleIndex

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#65] [Notes]
# ScriptReference/RaycastHit-triangleIndex

https://docs.unity3d.com/ScriptReference/RaycastHit-triangleIndex.html

**Comment by vertxxyz** [comment]
This will return `-1` if your Mesh Collider is marked as convex, as the convex mesh is not actually the same as the mesh assigned to the collider.
